routers/repo/pull_review.go

```diff
@@ -61,6 +61,56 @@ func CreateCodeComment(ctx *context.Context, form auth.CodeCommentForm) {
 	ctx.Redirect(comment.HTMLURL())
 }
 
+// UpdateResolveConversation add or remove an Conversation resolved mark
+func UpdateResolveConversation(ctx *context.Context) {
+	action := ctx.Query("action")
+	issueID := ctx.QueryInt64("issue_id")
+	commentID := ctx.QueryInt64("comment_id")
```

Comment:
The `issue_id` parameter is not actually required, IMHO. `comment_id` should suffice; then you can load the comment and perform `comment.LoadIssue()`. That's probably better, because the comment will most likely need the issue at some point, so we'd be saving one query to the database.